metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -132,6 +132,14 @@ class UnityCatalogSourceConfig(
 
     _metastore_id_pattern_removed = pydantic_removed_field("metastore_id_pattern")
 
+    catalogs: Optional[List[str]] = pydantic.Field(
+        default=None,
+        description=(
+            "Fixed list of catalogs to ingest."
+            " If not specified, catalogs will be ingested based on `catalog_pattern`."
```

Comment:
our allowdenypattern has a `is_fully_specified_allow_list` property that could be used as a shortcut and avoid requiring an additional config option